.travis.yml

```diff
@@ -1,24 +1,22 @@
+sudo: false
 language: perl
 matrix:
     fast_finish: true
 perl:
-    - "5.24"
-    - "5.22"
-    - "5.20"
-    - "5.18"
-    - "5.16"
-    - "5.14"
-    - "5.12"
+   - '5.24'
+   - '5.22'
+   - '5.20'
+   - '5.18'
+   - '5.16'
+   - '5.14'
```

Comment:
I'd recommend updating to

```yaml
perl:
   - "5.28"
   - "5.26"
   - "5.24"
   - "5.14"
```

As the last one is the earliest we can care to support, while the latest 3 are currently supported per https://metacpan.org/pod/perlpolicy#MAINTENANCE-AND-SUPPORT

Reply:
OK, should we also update this doc ?https://github.com/regentmarkets/cpan/wiki/Using-dzi
l

Reply:
I updated it ,seems traivs fail on 5.28

Reply:
t/00-compile.t ......... 1/2 Locale::Country will be removed from the Perl core distribution in the next major release
Locale::Codes will be removed from the Perl core distribution in the next major release. 

we got those waring for use perl 5.28, should we remove 5.28?